java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -0,0 +1,51 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.StreamAddOptions;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Stream Commands" group for standalone clients and
+ * cluster clients.
+ *
+ * @see <a href="https://redis.io/commands/?group=stream">Stream Commands</a>
+ */
+public interface StreamBaseCommands {
+
+    /**
+     * Adds an entry to the specified stream.
```

Comment:
Should we mention that this creates a new stream if is missing?

Reply:
I'm just following the existing node documentation. 
https://github.com/aws/glide-for-redis/blob/main/node/src/BaseClient.ts#L1275

Reply:
- [ ]  raise a separate PR to clean this up in both languages

Reply:
https://github.com/Bit-Quill/glide-for-redis/pull/186